crates/networking/p2p/rlpx/connection/server.rs

```diff
@@ -850,6 +911,16 @@ async fn handle_block_range_update(state: &mut Established) -> Result<(), RLPxEr
     }
 }
 
+async fn handle_send_ping(state: &mut Established) -> Result<(), RLPxError> {
+    match send(state, Message::Ping(PingMessage {})).await {
+        Ok(_) => {
+            log_peer_debug(&state.node, "Ping sent");
```

Comment:
Maybe logging each Ping is too much. The use of this function should be replaced by:
```
send(state, Message::Ping(PingMessage {})).await
```